src/pages/trade/ui/chart.tsx

```diff
@@ -71,7 +71,7 @@ const ChartData = observer(
     }, [chartElRef]);
 
     useEffect(() => {
-      if (chartRef.current && !isLoading) {
+      if (chartRef.current && !isLoading && candles) {
```

Comment:
On error, the casting on https://github.com/penumbra-zone/dex-explorer/pull/117/files#diff-c44d16aa7a593e9bbbb5d025963ae793cd30f7c22f703d4b5d4e195e766f20ebR83 causes a bug. Adding the guard here to prevent it.

Reply:
👍